packages/next-international/README.md

```diff
@@ -273,15 +266,102 @@ It's a simple wrapper function around other locales:
 ```ts
 // locales/fr.ts
 export default defineLocale({
-  'hello': 'Bonjour',
-  'welcome': 'Bonjour {name}!',
-})
+  hello: 'Bonjour',
+  welcome: 'Bonjour {name}!',
+});
 ```
 
 ### Use the types for my own library
 
 We also provide a separate package called [international-types](https://github.com/QuiiBz/next-international/tree/main/packages/international-types) that contains the utility types for next-international. You can build a library on top of it and get the same awesome type-safety.
 
+### Testing
+
+In case you want use tests with next-international you will need to create a custom render. The following example uses `@testing-library` and `Vitest`, but should work with `Jest` too.
```

Comment:
```suggestion
In case you want to make tests with next-international, you will need to create a custom render. The following example uses `@testing-library` and `Vitest`, but should work with `Jest` too.
```